src/com/lftechnology/java/training/sanish/application/component/EmployeeHelper.java

```diff
@@ -0,0 +1,416 @@
+package com.lftechnology.java.training.sanish.application.component;
+
+import com.lftechnology.java.training.sanish.application.model.dao.EmployeeDao;
+import com.lftechnology.java.training.sanish.application.model.dao.UserDao;
+import com.lftechnology.java.training.sanish.application.model.domain.Employee;
+import com.lftechnology.java.training.sanish.application.model.domain.User;
+import com.lftechnology.java.training.sanish.application.model.domain.UserEmployee;
+import com.lftechnology.java.training.sanish.application.utility.UserInput;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide functions of employee related
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class EmployeeHelper {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeHelper.class.getName());
+
+    private EmployeeHelper() {
+    }
+
+    /**
+     * Change user name of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param user         {@link User}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeUserName(Scanner inputScanner, User user) {
+        String message = "\n" + Constants.USER_NAME_LABEL + user.getUserName() + "\n>>" + Constants.CHANGE_USER_NAME_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String userName;
+            String SetStatement;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00117)